lib/pushmi_pullyu/cli.rb

```diff
@@ -89,6 +95,20 @@ def parse_options(argv)
         config.monitor = true
       end
 
+      opts.on('-rh', '--redis-host IP', 'Host IP of Redis instance to read from.'\
```

Comment:
Not a bad idea. I think this is fine for now. But If this option parser continues to grow in size, we might need to do something to keep this from growing ridiculously large. 

Whether that is to just move it into a separate class/method. Or better yet, reserve the command line options for just dealing with running the program and any configuration with talking to redis/solr/fedora/swift etc to live in a config yaml file. Which it's location will be defaulted to `config/pushmi_pullyu.yml` or `config/pushmi_pullyu.yml.erb` and can be override on the command line:

```ruby
        opts.on '-C', '--config PATH', "path to YAML config file" do |config_file|
          config.config_file = config_file
        end
```
Then after the options parser we can do a override the options with the config file

This also gives us the ability to override for different environments (staging/production) and much more power. Instead of passing 20 different options on the command line to `pushmi_pullyu`